cdap-common/src/main/java/io/cdap/cdap/common/operation/LongRunningOperationRequest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.common.operation;
```

Comment:
why is this in common?

Reply:
Any module can implement an operation so it has to be in a shared module. As this a CDAP internal API and not  I did not put it in `cdap-system-app-api` because my understanding was `cdap-system-app-api` should only external facing programatic APIs.